.changeset/eight-kids-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/backend": patch
```

Comment:
New feature or bug fix? 🤔 

```suggestion
"@clerk/backend": minor
```

Reply:
I'd argue that we were missing a field that should have been there [eight months ago](https://github.com/clerk/javascript/commit/0518331675ffb4d6c6830d79a1d61f9e4466773a#diff-b2a794b73af926a353a11d0af8f40c80c83296326bfba0a4667c36a355e230fcR174), so it feels like a bug rather than a feature.